README.md

```diff
@@ -1,8 +1,24 @@
-# Wiresmash
+# Wiresmash <!-- omit in toc -->
 
 Wiresmash is a Smash-like game currently under development by the iScsc dev team.
 
-## Currently
+## Table of content <!-- omit in toc -->
+
+- [Useful links](#useful-links)
+- [Current Features](#current-features)
+- [Launching](#launching)
+- [Building from source](#building-from-source)
+  - [Requirements](#requirements)
+  - [Building with CMake](#building-with-cmake)
+- [Future Plans](#future-plans)
+
+## Useful links
+
+- [The iscsc website](https://iscsc.fr/)
+- [How to contribute](CONTRIBUTING.md)
```

Comment:
this link is dead.
did you forget to add `CONTRIBUTING.md`?

Reply:
I was aware, and thus I should have added a "soon" near to it, but since it would have linked properly once a `CONTRIBUTING.md` would be in the repo, I thought it was fine.

Add one in [e4ecc74](https://github.com/iScsc/wiresmash/pull/21/commits/e4ecc74c946cbb6d474bc2494c025d4e5a300fbe) anyway

Reply:
that's fine :wink: